Graphics/Player judgment.lua

```diff
@@ -55,6 +72,16 @@ return Def.ActorFrame{
 					-- This technically causes a discrepency on the histogram, but it's likely okay.
 					if not IsW0Judgment(param, player) and not IsAutoplay(player) then
 						frame = 1
+						
+						for col,tapnote in pairs(param.Notes) do
+							local tnt = ToEnumShortString(tapnote:GetTapNoteType())
+							if tnt == "Tap" or tnt == "HoldHead" or tnt == "Lift" then
+								SCREENMAN:GetTopScreen()

```

Comment:
On the itgmania-beta branch we added a function to get the player such that it also works for edit mode. Can we do a couple of things (https://github.com/Simply-Love/Simply-Love-SM5/blob/itgmania-beta/Graphics/Player%20judgment.lua#L6-L48)

Can we copy that function in this branch as well and use it?